### PR TITLE
content: add chat links (gitter/matrix) for ethereum-solidity

### LINKED ISSUE #4753: Feature: Add community links(gitter/matrix) for ethereum-solidity
**Is your feature request related to a problem? Please describe.**

Community links for Solidity(https://github.com/ethereum/solidity) to be added here https://ethereum.org/en/community/online/

- Gitter: https://gitter.im/ethereum/solidity
- Matrix: https://matrix.to/#/#ethereum_solidity:gitter.im

Community Links Source: https://forum.soliditylang.org/

### PR DESCRIPTION
**Pull request by ErikBjare**
Adds chat links for ethereum-solidity.

## Description

 - [x] ~~Not sure if `gitter` and `matrix` are valid `socialIcon` options. Should perhaps be changed to `webpage`.~~
 - [ ] Maybe one of them is enough? (they seem to be bridged anyway)

## Related Issue

Fixes #4753

### CODE REVIEW

File: src/content/community/online/index.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
 <SocialListItem socialIcon="discord"><Link to="https://discord.io/ethstaker">EthStaker Discord</Link> <i>- community oriented around offering project management support to Ethereum development</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/CetY6Y4">Ethereum.org website team</Link> <i>- stop by and chat ethereum.org web development and design with the team and folks from the community</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/ZH5aXDgWEU">Web3 University</Link> <i>- community focused on learning web3 development </i></SocialListItem>
+<SocialListItem socialIcon="gitter"><Link to="gitter.im/ethereum/solidity">Solidity Gitter</Link> <i>- chat for solidity development (Gitter)</i></SocialListItem>
```
Comment (ErikBjare) [inline src/content/community/online/index.md:32]:
```suggestion
<SocialListItem socialIcon="website"><Link to="gitter.im/ethereum/solidity">Solidity Gitter</Link> <i>- chat for solidity development (Gitter)</i></SocialListItem>
```

File: src/content/community/online/index.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
...
+<SocialListItem socialIcon="matrix"><Link to="matrix.to/#/#ethereum_solidity:gitter.im">Solidity Gitter</Link> <i>- chat for solidity development (Matrix)</i></SocialListItem>
```
Comment (ErikBjare) [inline src/content/community/online/index.md:33]:
```suggestion
<SocialListItem socialIcon="website"><Link to="matrix.to/#/#ethereum_solidity:gitter.im">Solidity Gitter</Link> <i>- chat for solidity development (Matrix)</i></SocialListItem>
```

File: src/content/community/online/index.md [hunk 3]
```diff
@@ -29,6 +29,9 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
...
+
```
Comment (ErikBjare) [inline src/content/community/online/index.md:34]:
```suggestion
```

File: src/content/community/online/index.md [hunk 4]
```diff
@@ -29,6 +29,8 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
 <SocialListItem socialIcon="discord"><Link to="https://discord.io/ethstaker">EthStaker Discord</Link> <i>- community oriented around offering project management support to Ethereum development</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/CetY6Y4">Ethereum.org website team</Link> <i>- stop by and chat ethereum.org web development and design with the team and folks from the community</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/ZH5aXDgWEU">Web3 University</Link> <i>- community focused on learning web3 development </i></SocialListItem>
+<SocialListItem socialIcon="website"><Link to="gitter.im/ethereum/solidity">Solidity Gitter</Link> <i>- chat for solidity development (Gitter)</i></SocialListItem>
```
Comment (minimalsm) [inline src/content/community/online/index.md:32]: I think you're trying to add the globe 🌐? We use `webpage` for the socialIcon id.

```suggestion
<SocialListItem socialIcon="webpage"><Link to="gitter.im/ethereum/solidity">Solidity Gitter</Link> <i>- chat for solidity development (Gitter)</i></SocialListItem>
```

File: src/content/community/online/index.md [hunk 5]
```diff
@@ -29,6 +29,8 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
...
+<SocialListItem socialIcon="website"><Link to="matrix.to/#/#ethereum_solidity:gitter.im">Solidity Matrix</Link> <i>- chat for solidity development (Matrix)</i></SocialListItem>
```
Comment (minimalsm) [inline src/content/community/online/index.md:33]:
```suggestion
<SocialListItem socialIcon="webpage"><Link to="matrix.to/#/#ethereum_solidity:gitter.im">Solidity Matrix</Link> <i>- chat for solidity development (Matrix)</i></SocialListItem>
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks for the PR @ErikBjare 

Made a tiny change to fix the social icon :-) just waiting on this building and I think we're good to merge.

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for the PR @ErikBjare 🤘

Be sure to [[join the discord](https://discord.gg/CetY6Y4)](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [[claim your POAP](https://ethereum.org/en/contributing/#poap)](https://ethereum.org/en/contributing/#poap) also 🥳! We're currently working on our 2022 POAPs and they'll be released soon.

**Comment by minimalsm** [comment]
@all-contributors please add @ErikBjare for docs, code

**Pull request merged**